packages/suite-web/e2e/downloads/Bitcoin_10_20240829T072722.csv

```diff
@@ -0,0 +1,19 @@
+Timestamp;Date;Time;Type;Transaction ID;Fee;Fee unit;Address;Label;Amount;Amount unit;Fiat (USD);Other
```

Comment:
this is some leftover?

Reply:
I'll ask @jvaclavik what's that I am really not sure where this came from.  THanks

Reply:
I have no idea